IBX-11100: Hybrid tracking described in Dev-doc (#3250)
ezsystems/developer-documentation: docs/recommendations/raptor_integration/connector_installation_configuration.md

- `tracking_type` - defines how user events are sent to the tracking API. Default value: `client`. Possible values:
    - `client` - tracking is executed in the browser using JavaScript snippets generated by the [Twig functions](recommendations_twig_functions.md) and included in the templates. This approach may be blocked by ad blockers.
    - `server` - tracking is handled on the backend, with events sent directly to the tracking API. It's not affected by ad blockers.

>     - `hybrid` - loads a local, first-party tracking JavaScript provided by the DXP instance, instead of the Raptor SaaS JavaScript.

Reviewer: ```suggestion
    - `hybrid` - loads a local, first-party tracking JavaScript provided by the DXP instance, instead of the Raptor SaaS JavaScript. For more information, see [Hybrid tracking](hybrid_tracking.md).
```

>     - `hybrid` - loads a local, first-party tracking JavaScript provided by the DXP instance, instead of the Raptor SaaS JavaScript. For more information, see [Hybrid tracking](hybrid_tracking.md).

Reviewer 2: "local" sounds a bit weird. I guess it means in the browser.
I would try to have the same style than the two other possibilities.
```suggestion
    - `hybrid` - tracking is executed in the browser by a first-party JavaScript provided by the DXP instead of Raptor and then forwarded by the DXP server to the Raptor SaaS. For more information, see [Hybrid tracking](hybrid_tracking.md).
```

- `strict_exceptions` – when enabled, tracking exceptions are thrown instead of being silently handled. Default value: `%kernel.debug%`.

> - `hybrid_tracking_proxy_path` - by default, it's set to `/raptor/track`, the client side shim sends tracking events to this same-origin endpoint, which forwards them to Raptor asynchronously.

Reviewer: ```suggestion
- `hybrid_tracking_proxy_path` - by default, it's set to `/raptor/track`. The client side shim sends tracking events to this same-origin endpoint, which forwards them to Raptor asynchronously.
```

> - `hybrid_tracking_proxy_path` - by default, it's set to `/raptor/track`. The client side shim sends tracking events to this same-origin endpoint, which forwards them to Raptor asynchronously.

Reviewer: ```suggestion
- `hybrid_tracking_proxy_path` - by default, it's set to `/raptor/track`. The client-side shim sends tracking events to this same-origin endpoint, which forwards them to Raptor asynchronously.
```
For consistency with the changes from github.com/ibexa/documentation-developer/pull/3250/commits/c0e61f0496b8cc0b3b9689fccbd5c3836777b1a4 (it's a good change)
